service/frontend/templates/clusterredirection.tmpl

```diff
@@ -112,7 +113,8 @@ func (handler *clusterRedirectionHandler) {{$method.Declaration}} {
 		case targetDC == handler.currentClusterName:
 			{{$method.ResultsNames}} = handler.frontendHandler.{{$method.Call}}
 		default:
-			remoteClient, err := handler.GetRemoteFrontendClient(targetDC)
+			var remoteClient frontend.Client
+			remoteClient, err = handler.GetRemoteFrontendClient(targetDC)
```

Comment:
can you update unit test for at least one of the methods to cover the problematic case that is being fixed?